sentry/src/event_reducer.rs

```diff
@@ -101,10 +101,10 @@ mod test {
             ad_slot: None,
             referrer: None,
         };
-
+        let payout = Some((IDS["publisher"], BigNum::from(1)));
         for i in 0..101 {
-            reduce(&channel, &mut event_aggr, &event, &None)
-                .expect(&format!("Should be able to reduce event #{}", i));
```

Comment:
this is actually the same. Expect will unwrap or it will panic, no need to change this.